web-dev/src/App.js

```diff
@@ -20,19 +20,24 @@ import UserComponent from "./components/RegistrationPage/UserComponent";
 function App() {
   return (
     <BrowserRouter>
-      <div className="flex-container">
+      <div className="container-fluid">
```

Comment:
What's the difference between container-fluid and flex-container?

Reply:
Honestly, I have no idea why Viet use "flex-container". I find that in some of his code but no CSS reference to it. 
But I can answer for container-fluid vs container. Please visit the search or one of the detail screens and adjust the browser width. You will see that the navbar (container-fluid) stretches fluidly as the browser changes, but my components change on breakpoint. I think you can switch back to container if you like. I personally like container for my part that why I force it.